install/first_google_boot.sh

```diff
@@ -249,26 +249,43 @@ function extract_spinnaker_gcr_credentials() {
   mkdir -p $(dirname $config_path)
   chown -R spinnaker:spinnaker $(dirname $config_path)
 
+  local gcr_enabled=$(get_instance_metadata_attribute "gcr_enabled")
   local gcr_account=$(get_instance_metadata_attribute "gcr_account")
-  if [ -n "$gcr_account" ]; then
-    # This downloads the JSON key for the specified service account
+
+  if [ -n "$gcr_enabled" ] || [ -n "$gcr_account" ]; then
+    echo "GCR enabled"
+
```

Comment:
The idea is setting a gcr_account implicilty means you intend to use GCR. Not setting gcr_account, but setting gcr_enabled means you use the default compute account.

gcr_enabled = ! [ -z gcr_account ] only triggers when both flags are set, which is not the behavior we want.